### ISSUE TITLE
Suggestion: colored version mismatches, and an update log

### ISSUE THREAD

**Issue by Cheremenin** [RimSort/RimSort#316]
So I've recently switched from RimPy seeing as it apparently is becoming abandonware. So far I really love RimSort, thank you for creating it! I think it sorts in a better way (more up-to-date DB?), and also the texture optimizer seems to be faster than the one included in RimPy, and more automated. But I'm still really missing a couple of things that were extremely nifty.

**Is your feature request related to a problem? Please describe.**
Not really a problem per se. More like an implementation of the only features I actually really miss from RimPy.

**Describe the solution you'd like**
RimPy had two amazing things I've considered a lifesaver and a killer feature - the first is simply the colored names of mods that change depending on whether the mod is working for your installed game version, or not. The mods that are safe to add are the normal white color, questionable or incompatible mods are red. Very at a glance, useful and visually appealing.

The second thing is the update tracker. I take a quick look at it and I just immediately know which one of my favorite mods have recently got an update, or are now compatible with 1.5, so I can add them to my game. See, I have quite a lot of subbed mods, but I only actively use about 200, so managing them all can get a little tedious, but RimPy always helped with these two things immensely.

I'm currently firing up RimPy from time to time to check the update log cause I just find it extremely useful, but otherwise I'm now a RimSort user for the most part. I hope eventually similar features appear in it, so I can switch completely.

**Comment by Limfjorden** [comment]
The colored names are implemented now.

**Comment by LionelColaso** [comment]
Implemented in #821
